frontend/src/metabase/dashboard/components/DashCard/utils.ts

```diff
@@ -16,3 +32,67 @@ export function shouldShowParameterMapper({
     !(display && VIZ_WITH_CUSTOM_MAPPING_UI.includes(display))
   );
 }
+
+type ParameterDimensionTarget = ["dimension", LocalFieldReference];
```

Comment:
@ranquild @kulyk 

we already have the type ParameterDimensionTarget like

```
export type ParameterDimensionTarget = [
  "dimension",
  ["field", FieldId, ReferenceOptions | null] | ["template-tag", string],
];
```

am I right that we expected only the field here?

Reply:
nvm, provided a fix here [92b16cb](https://github.com/metabase/metabase/pull/38545/commits/92b16cb5d5a682344b04e121b89b2c6a4a3616af)